pkg/service/admission/externalip_admission.go

```diff
@@ -97,7 +115,7 @@ func (r *externalIPRanger) Admit(a kadmission.Attributes) error {
 				errs = append(errs, field.Forbidden(field.NewPath("spec", "externalIPs").Index(i), "externalIPs must be a valid address"))
 				continue
 			}
-			if NetworkSlice(r.reject).Contains(ip) || !NetworkSlice(r.admit).Contains(ip) {
+			if (NetworkSlice(r.reject).Contains(ip) || !NetworkSlice(r.admit).Contains(ip)) && s != ingressIP {
```

Comment:
Would suggest turning s != ingressIP into a boolean `ingressIPChanged` and referencing that instead.  Makes it easier to read.